tcl/tson.tcl

```diff
@@ -119,3 +119,19 @@ proc qc::tson2xml { tson } {
 	}
     }
 }
+
+proc qc::tson_get {tson args} {
+    #| Returns the value at the specified path in the TSON.
+    #| Requires PostgreSQL 9.3 or later.
+
+    # Construct a PostgreSQL array literal from the path.
+    set path "\{[join $args ","]\}"
```

Comment:
This is a structured string.
Do we have a library tool to construct a postgresql array literal ?